cime_config/config_compsets.xml

```diff
@@ -77,6 +77,12 @@
      <lname>1850_DATM%CRUv7_CLM50%BGC-CROP_CICE_POP2_MOSART_CISM2%GRIS-EVOLVE_SWAV</lname>
   </compset>
 
+  <!-- fully coupled without hybrid coordinate in BLOM for testing-->
+  <compset>
+    <alias>N1850nohyb</alias>
```

Comment:
```suggestion
    <alias>N2N1850</alias>
```
In the BLOM `config_compsets.xml` file we have named the compsets using isopycninc vertical grids with a prefix "N2..." to indicate that this is equivalent to the NorESM2 compset alias. Perhaps not the best naming convention, but we can try to be consistent.